system/lib/gl.c

```diff
@@ -1560,174 +1560,240 @@ void* emscripten_GetProcAddress(const char *name_) {
+  if (!strcmp(name, "glBlendEquation")) return emscripten_glBlendEquation;
+  if (!strcmp(name, "glIsEnabled")) return emscripten_glIsEnabled;
+  if (!strcmp(name, "glBlendFunc")) return emscripten_glBlendFunc;
+  if (!strcmp(name, "glBlendEquationSeparate")) return emscripten_glBlendEquationSeparate;
+  if (!strcmp(name, "glDepthRange")) return emscripten_glDepthRange;
+  if (!strcmp(name, "glDepthRangef")) return emscripten_glDepthRangef;
+  if (!strcmp(name, "glStencilMaskSeparate")) return emscripten_glStencilMaskSeparate;
+  if (!strcmp(name, "glHint")) return emscripten_glHint;
+  if (!strcmp(name, "glPolygonOffset")) return emscripten_glPolygonOffset;
+  if (!strcmp(name, "glVertexAttrib1f")) return emscripten_glVertexAttrib1f;
+  if (!strcmp(name, "glSampleCoverage")) return emscripten_glSampleCoverage;
+  if (!strcmp(name, "glTexParameteri")) return emscripten_glTexParameteri;
+  if (!strcmp(name, "glTexParameterf")) return emscripten_glTexParameterf;
+  if (!strcmp(name, "glVertexAttrib2f")) return emscripten_glVertexAttrib2f;
+  if (!strcmp(name, "glStencilFunc")) return emscripten_glStencilFunc;
+  if (!strcmp(name, "glStencilOp")) return emscripten_glStencilOp;
+  if (!strcmp(name, "glViewport")) return emscripten_glViewport;
+  if (!strcmp(name, "glClearColor")) return emscripten_glClearColor;
+  if (!strcmp(name, "glScissor")) return emscripten_glScissor;
+  if (!strcmp(name, "glVertexAttrib3f")) return emscripten_glVertexAttrib3f;
+  if (!strcmp(name, "glColorMask")) return emscripten_glColorMask;
+  if (!strcmp(name, "glRenderbufferStorage")) return emscripten_glRenderbufferStorage;
+  if (!strcmp(name, "glBlendFuncSeparate")) return emscripten_glBlendFuncSeparate;
+  if (!strcmp(name, "glBlendColor")) return emscripten_glBlendColor;
+  if (!strcmp(name, "glStencilFuncSeparate")) return emscripten_glStencilFuncSeparate;
+  if (!strcmp(name, "glStencilOpSeparate")) return emscripten_glStencilOpSeparate;
+  if (!strcmp(name, "glVertexAttrib4f")) return emscripten_glVertexAttrib4f;
+  if (!strcmp(name, "glCopyTexImage2D")) return emscripten_glCopyTexImage2D;
+  if (!strcmp(name, "glCopyTexSubImage2D")) return emscripten_glCopyTexSubImage2D;
+  if (!strcmp(name, "glDrawBuffers")) return emscripten_glDrawBuffers;
+  // UBO
+  if (!strcmp(name, "glGetUniformIndices")) return emscripten_glGetUniformIndices;
+  if (!strcmp(name, "glGetActiveUniformsiv")) return emscripten_glGetActiveUniformsiv;
+  if (!strcmp(name, "glGetActiveUniformName")) return emscripten_glGetActiveUniformName;
+  if (!strcmp(name, "glGetUniformBlockIndex")) return emscripten_glGetUniformBlockIndex;
+  if (!strcmp(name, "glGetActiveUniformBlockiv")) return emscripten_glGetActiveUniformBlockiv;
+  if (!strcmp(name, "glGetActiveUniformBlockName")) return emscripten_glGetActiveUniformBlockName;
+  if (!strcmp(name, "glUniformBlockBinding")) return emscripten_glUniformBlockBinding;
+  // GLES 3.0 functions
```

Comment:
I assume all the new lines are from here, and above this it was just removing the `else`? (which I agree is a nice change)

Reply:
yes, exactly